src/script/components/MessagesList/Message/MessageWrapper.tsx

```diff
@@ -234,6 +235,9 @@ export const MessageWrapper: React.FC<MessageParams & {hasMarker: boolean; isMes
   if (message.isCallTimeout()) {
     return <CallTimeoutMessage message={message} />;
   }
+  if (message.isFailedToAddUsersMessage()) {
+    return <FailedToAddUsersMessage message={message} />;
```

Comment:
I think there's an issue with putting that warning in the message wrapper.

The mocks put the warning between the added users and the following system messages (read receipts etc)

putting it here would put here after every system message and add a timestamp.

`MemberMessage.tsx` seems to be where those system message live, wdyt?